RELEASE.md

```diff
@@ -2,14 +2,14 @@
 
 _What is being released?_
 
-This release updates the `rosetta-bundle` dependencies.
+This release updates the `rosetta-dsl` and `rosetta-bundle` dependencies.
 
 Version updates include:
-
-- `rosetta-bundle` 10.9.3: this release adds mapping support for XSD substitution groups, which fixes the issue related to the mapping of FpML oilPhysicalLeg xml elements.
+- `rosetta-dsl` 9.5.0: For further details see DSL release notes: https://github.com/REGnosys/rosetta-dsl/releases/tag/9.5.0.
+- `rosetta-bundle` 10.10.3
```

Comment:
The previous release note is unreleased, so these 2 release notes should be merged